README.md

```diff
@@ -101,6 +101,9 @@ require'treesitter-context'.setup{
         yaml = {
             'block_mapping_pair',
         },
+        python = {
+            'def',
```

Comment:
python functions are already covered by the `function` pattern in the default config. 
In my tests this also works.

Reply:
or was something else missing here?

Reply:
My bad, I'm new to nvim/treesitter. Had some issues getting everything setup properly and thought 'def' weren't working.

You're correct, the default config works with Python 'def'.